raft.go

```diff
@@ -927,6 +940,23 @@ func (r *Raft) appendEntries(rpc RPC, a *AppendEntriesRequest) {
 	return
 }
 
+// checkAndProcessConfigurationLog takes a log entry and updates the latest
+// configuration if the entry results in a new configuration. This must only be
+// called from the main thread.
```

Comment:
Ah it needs the transport to support the old methods, not worth de-membering. When we kill the support for the old protocol we can nuke this fn and just inline the 4 lines that will matter at that point.